pr-data.csv

```diff
@@ -4370,7 +4370,7 @@ https://github.com/GoogleCloudPlatform/DataflowTemplates,5094c7b39de511c9ed441d9
 https://github.com/GoogleCloudPlatform/DataflowTemplates,cc17ed154d8c3c385ab3f5a3653a6e4c02539bfa,v1,com.google.cloud.teleport.templates.common.BigQueryConvertersTest.testFailsafeJsonToTableRowValidInput,ID,,,
 https://github.com/GoogleCloudPlatform/DataflowTemplates,5094c7b39de511c9ed441d9fde28553a88f68e4b,.,com.google.cloud.teleport.templates.common.DatastoreConvertersTest.testCheckNoKeyAllInvalid,ID,,,
 https://github.com/GoogleCloudPlatform/DataflowTemplates,5094c7b39de511c9ed441d9fde28553a88f68e4b,.,com.google.cloud.teleport.templates.common.DatastoreConvertersTest.testCheckNoKeyBothCorrectAndInvalid,ID,,,
-https://github.com/GoogleCloudPlatform/DataflowTemplates,5094c7b39de511c9ed441d9fde28553a88f68e4b,.,com.google.cloud.teleport.templates.common.DatastoreConvertersTest.testCheckSameKey,ID,,,
+https://github.com/GoogleCloudPlatform/DataflowTemplates,5094c7b39de511c9ed441d9fde28553a88f68e4b,.,com.google.cloud.teleport.templates.common.DatastoreConvertersTest.testCheckSameKey,ID,Opened,https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1924,https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1924
```

Comment:
Remove the "note" (last column) for an `Opened` PR. The link to the PR is all we need.

Reply:
Thanks for the comment. Done.